### PR TITLE
Multi-Cart improvement 

### PR DESCRIPTION
**Pull request by yasiloghmani**
Multi-Cart improvement 

Description:

- Add multi-cart pagination

Linting:

- [X] No linting errors

Tests:

- [X] Manual tests
- [ ] Accessibility tests (no new react-axe errors in console)

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "3.0.0",
   "private": true,
   "dependencies": {
-    "@moltin/sdk": "5.5.0",
+    "@moltin/sdk": "6.1.0",
```
Comment (shaunmaharaj) [inline package.json:6]: need to update to 6.1.2

File: src/CartItemList.tsx [hunk 2]
```diff
@@ -120,29 +136,35 @@ export const CartItemList: React.FC<CartItemListParams> = (props) => {
...
-          )}
-          {partialAddMessage &&  (
-            <div className="partialadd__alertMessage">
-              <p>{partialAddMessage}</p>
-              <CloseIcon onClick={() => setPartialAddMessage("")}/>
-            </div>
-          )}
+      <div className="partialadd">
+        {addedtItem &&  ( <div className="partialadd__confirmationmessage">
+          <p>{addedtItem}  items have been added to the cart</p>
```
Comment (shaunmaharaj) [inline src/CartItemList.tsx:156]: This string should be in our localization file

File: src/CartItemList.tsx [hunk 3]
```diff
@@ -112,6 +112,22 @@ export const CartItemList: React.FC<CartItemListParams> = (props) => {
...
       }, 4000);
   }, [showUpdateCartAlert])
+  
+  useEffect(() => {
+    window.scrollTo(0,0);
+  })
+
+  useEffect(() => {
+    const el = document.querySelector('.cartmodal__content');
+    console.log(el)
```
Comment (shaunmaharaj) [inline src/CartItemList.tsx:122]: console.log leftover

File: src/CartsList.tsx [hunk 4]
```diff
@@ -93,6 +94,19 @@ export  const CartsList: React.FC<CartsListParams> = (props) => {
         setShowCreateCartAlert(false)
       }, 4000);
   }, [showCreateCartAlert]);
+  
+
+  useEffect(() => {
+    const el = document.querySelector('.cartmodal__content');
+    console.log(el)
```
Comment (shaunmaharaj) [inline src/CartsList.tsx:101]: console.log leftover

**Comment by yasiloghmani** [comment]
Yasaman Loghmani
Software Developer
________________________________

**Review by shaunmaharaj — APPROVED** [review]

**Pull request merged**